R/rsa.r

```diff
@@ -0,0 +1,16 @@
+#' RSA SHA1
+#'
+#' As described in \url{http://tools.ietf.org/html/rfc3447}.
+#'
+#' @param key secret key
+#' @param string data to securely hash
+#' @keywords internal
+#' @export
+rsa_sha1 <- function(key, string) {
```

Comment:
I think this should go in the same file (and possibly be documented with) `hmac_sha1`